packages/@atjson/document/src/annotation.ts

```diff
@@ -197,7 +198,7 @@ export abstract class Annotation<Attributes = {}> {
    *     are applied to the document including annotations.
    */
   handleChange(change: Change) {
-    if (change.type === "insertion") {
+    if (change.type === ChangeType.insertion) {
```

Comment:
Unrelated to the boundary work, but I was in here anyway, figured we should enum this ...

Reply:
I'm a bit hesitant to do this, because the design fo the change events was such that it could be sent to an operational transform server or a CRDT server. @blaine can possibly provide more context here.

Reply:
Fair enough, happy to revert that bit

Reply:
Yeah; I'm not sure how much it matters either way and what it'd look like exactly in terms of integrating into a CRDT world. Whether it's a string or an enum is probably irrelevant, so long as we don't start making changes to the document implicit or otherwise happen outside of this change handler flow.

Reply:
I'm just going to revert this bit regardless, unnecessary distraction from the primary focus of the PR